docs/website/docs/plus/core-concepts/cache.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "🧪 Cache"
+description: Execute data transformations in your local cache
+keywords: ["dlt+", "cache", "transformations"]
+---  
+  
+import Link from '../../_plus_admonition.md';
+
+<Link/>
+
+:::note
+🚧 This feature is under development. Interested in becoming an early tester? [Join dlt+ early access](https://info.dlthub.com/waiting-list)
+:::
+The dlt+ Cache is a temporary local storage created by dlt+ to enhance development workflows. It allows you to efficiently run local transformations, materialize dbt models, and test your queries before deploying them to production. 
+
+## How it works
+
+The dlt+ Cache is a powerful tool that enables users to shift parts of their data workflows earlier in the development process. Its primary use case today is [running transformations locally](../features/transformations/index.md), but we plan to support more use cases and workflows in the future.
+
+You specify which datasets you want to pass to the cache in your dlt manifest file (`dlt.yml`). The cache automatically discovers the source schema from the data and runs your transformations using the cache and duckdb as a query engine. Currently you can define your transformations in dbt or Python (pands, arrows, polars, etc.). After running your transformations, the cache will sync the results to the output dataset in your destination. Output schema is also automatically discovered (when not explicitly declared).
+
+
+## Why you should use it
+
+Using local cache for transformations provides several advantages, like:
```

Comment:
I also agree we should formulate clearer what the cache is, I started writing a technical explanation, I will post this later today.